docs/volume_annotation/tools.md

```
@@ -1,64 +1,129 @@
+- Relabeling segments with different IDs/colors
+- Quick corrections of small areas
+- The fill behavior can be modified using the 2D/3D fill modifiers (see below).
+- This tool only takes existing labels into account and does not look at any other underlying (microscopy) layers. Have a look at the [quick-select-tool](./quick_select.md) for quickly annotating new structures.
+
+![Segment Picker Tool](../ui/images/segment-picker-tool.jpg){align=left width="60"} 
+**Segment Picker**: Click any segment to use its label ID as the active segment ID and keep annotating with that ID. This is alternative to selecting the segment ID from the [Segments list](./segments_list.md) sidebar or context menu.
+
+![Quick Select Tool](../ui/images/quickselect-tool.jpg){align=left width="60"} 
+**Quick Select**: Automatically annotate segments using either:
+
+  - Threshold-based selection mode
```

Markdownlint (0.37.0)

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)